recipes/elk/webview.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (Ferdium) => {
+  const path = require('path');
```

Comment:
Please use the correct code from the template. This will not work due to the security checks.